lib/ASRouter.jsm

```diff
@@ -802,6 +802,11 @@ class _ASRouter {
       case "IMPRESSION":
         await this.addImpression(action.data);
         break;
+      case "TELEMETRY":
```

Comment:
Should this maybe be specific to the doorhanger, like DOORHANGER_TELEMETRY or something? also might be worth noting here or in the docs that this does not include a session id (since it's not tied to a new tab session at all)